gaurangrshah/transparency-ui
# resolved outadated packages issue

### the issue was regarding semver 

* semver was unable to find outdated packages 

* it seems normalization was required for installedPackages

* how to reproduce the issue 
 * add ```react: "^18.6.0"``` in REQUIRED_PACKAGES
 * make sure you have a lower version of react in package.json
 * then run these commands
 ```pnpm build && pnpm cli init```
